be/CMakeLists.txt

```diff
@@ -585,6 +585,7 @@ if (${MAKE_TEST} STREQUAL "ON")
     add_subdirectory(${TEST_DIR}/exprs)
     add_subdirectory(${TEST_DIR}/runtime)
     add_subdirectory(${TEST_DIR}/http)
+    add_subdirectory(${TEST_DIR}/aes)
```

Comment:
I think we don't need this 'aes' directory

you can put aes.h and aes.cpp to be/src/util, and rename these to aes_util.h??